tests/routing/cluster/decommission/awareness.yaml

```diff
@@ -0,0 +1,45 @@
+$schema: ../../../../json_schemas/test_story.schema.yaml
+
+description: Test cluster routing settings.
+version: '>= 2.16'
+prologues:
+  - id: weights
+    path: /_cluster/routing/awareness/zone/weights
+    method: GET
+    output:
+      version: payload._version ? -1
```

Comment:
This works fine for simple default values, but not scalable for more complex values, and it's not obviously if `-1` is a number or a string, or if `true` is a truthful bool or `"true"`. To accommodate this, we should allow the output definition to be either a string (which holds the retrieval path from the response_payload like how it's currently used), or an object:
```yaml
output:
  version:
    path: payload._version
    default: -1
```
This also makes the parsing logic a lot simpler.

Side tracking: does `payload._version` hint that the output values can come from something outside of the response payload?

Reply:
Yeah, not sure why I didn't do this from the start, thanks.

> does payload._version hint that the output values can come from something outside of the response payload?

Unfortunately not. If you assumed that the cluster was just started that the value is always -1, but the problem is that when testing locally these tests would fail on the second run.